Source/Core/Core/HW/Wiimote.cpp

```diff
@@ -182,24 +182,26 @@ void InterruptChannel(int number, u16 channel_id, const void* data, u32 size)
 
 bool ButtonPressed(int number)
 {
+  const WiimoteSource source = WiimoteReal::GetWiimoteSource(number);
+
   if (s_last_connect_request_counter[number] > 0)
   {
     --s_last_connect_request_counter[number];
-    if (g_wiimote_sources[number] && NetPlay::IsNetPlayRunning())
+    if (source != WiimoteSource::WIIMOTE_SRC_NONE && NetPlay::IsNetPlayRunning())
       Wiimote::NetPlay_GetButtonPress(number, false);
     return false;
   }
 
   bool button_pressed = false;
 
-  if (WIIMOTE_SRC_EMU & g_wiimote_sources[number])
+  if (WIIMOTE_SRC_EMU == source)
```

Comment:
`if (source == WIIMOTE_SRC_EMU)`
(I think we don't use Yoda conditions :P)

Reply:
Yoda conditional it already was.
I think there are still quite a few of them. I can change them if we really want.